derive/src/lib.rs

```diff
@@ -23,7 +32,43 @@ macro_rules! ident {
     };
 }
 
-fn expand_contract(input: LitStr) -> Result<TokenStream> {
+struct ContractArgs {
+    artifact_path: LitStr,
+    runtime_crate: Option<Ident>,
+}
+
+impl Parse for ContractArgs {
+    fn parse(input: ParseStream) -> ParseResult<Self> {
+        let mut result = ContractArgs {
+            artifact_path: input.parse()?,
+            runtime_crate: None,
+        };
+
+        while !input.is_empty() {
```

Comment:
Do we really need a while loop here? Can there be multiple crate as definitions in one line and do we really want to take the last then?

Reply:
No we don't need a while loop. The reason I implemented it that way is we have open cases for adding code generation options and will need it eventually. See #10 for that issue.